src/v2/view-builder/ViewFactory.js

```diff
@@ -230,7 +230,7 @@ const VIEWS_MAPPING = {
   },
   [RemediationForms.AUTHENTICATOR_VERIFICATION_DATA]: {
     [AUTHENTICATOR_KEY.PHONE]: ChallengeAuthenticatorDataPhoneView,
-    [AUTHENTICATOR_KEY.OV]: ChallengeAuthenticatorDataOktaVerifyView,
+    [AUTHENTICATOR_KEY.OV]: ChallengeAuthenticatorOktaVerifyView,
```

Comment:
ideally so we follow the name conventions that it seems we already have, this new file should have part of the remediation form in the name "ChallengeAuthenticatorData*", so can we rename it to either "ChallengeAuthenticatorDataOktaVerifyBaseView" or "ChallengeAuthenticatorDataOktaVerifyMasterView"?

or if you make the change for the other comments I made about exporting Body in the other 2 files, then this can actually just be named "ChallengeAuthenticatorDataOktaVerifyView" (as it is right now)